LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
If that's the case, then I'm fine making this AGPL. I've mostly heard the opposite, though - most organizations want the option to close source their changes.